### DISCUSSION TITLE
EthereumTransaction Zero HAPI Cost On Success

### DISCUSSION THREAD

**Discussion by Nana-EC** [hiero-ledger/hiero-improvement-proposals#1083] [Ideas]
**Context**
On Ethereum and other EVM chains  execution costs are fully captured in gas. 
On Hedera it was decided to charge a fee for `EthereumTransaction` HAPI submissions (just like all other HAPI transactions).

**Issue**
Operators of JSON-RPC relays have to pay the cost of the hardware, **and** they also have to pay a per-transaction fee of $0.0001. This is different from other networks, where a JSON-RPC relay operator only has to pay for the cost of the hardware, and all fees related to the transaction are paid by the sender of the transaction. This effectively forces relay operators on Hedera to adopt a unique business model that is more expensive for users, often in the form of different plans, which makes using Hedera more cumbersome for users than for other networks.

**Affected Users**
All relay operators and users of those relays.

**Solution**
The purpose of the $0.0001 fee was to defend the network against a malicious JSON-RPC relay by causing it to pay for badly formed transactions (or transactions where the sender cannot pay for it). However this fee is **not** needed for successful transactions, because the intrinsic gas fee is sufficient. Therefore, we propose to only charge the $0.0001 fee to a relay if the relay sends a transaction that cannot be paid for by the sender (either because the bytes are malformed, the user payload fails signature verification, the user account doesn't exist, the user doesn't have sufficient funds, or some other due-diligence failure on the part of the relay).

**Acceptance Cases**
    - If a user submits a valid ETH TX the relay operator will pay $0 and the user will be charged the appropriate gas
    - If a relay submits a transaction that fails one of the due diligence checks, they must pay the $0.0001 fee.

**Comment by Neurone** [comment]
Link to the [PR](https://github.com/hashgraph/hedera-improvement-proposal/pull/1084)
